lib/data_update_scripts/20220121114445_stripping_html_tags_from_tag_short_summary.rb

```diff
@@ -0,0 +1,11 @@
+module DataUpdateScripts
+  class StrippingHtmlTagsFromTagShortSummary
+    def run
+      Tag.where("short_summary LIKE '%<%'").find_each do |tag|
+        # Choosing to skip validations and mimic the newly added before_validation behavior.
+        new_short_summary = ActionController::Base.helpers.strip_tags(tag.short_summary)
+        tag.update_columns(short_summary: new_short_summary)
```

Comment:
The rationale behind `update_columns` here (always worth questioning when you see it) that having modified one field, and being otherwise unaware of the validity of the tag, you want to bypass validation, and update the row regardless?

Otherwise `tag.save` would validate, calling the tidy up method and have the same effect, but only if the objects were otherwise valid - and there's no guarantee of that.

Reply:
Great reading note, explaining in more verbose detail what I wanted to accomplish.

Also, I'd love to avoid a cascade of things that happen when we save a tag.